.github/workflows/dotnetcore.yml

```diff
@@ -52,4 +52,7 @@ jobs:
       - name: Run dotnet format
         uses: jfversluis/dotnet-format@v1.0.5
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          action: fix
```

Comment:
you can remove this or change it to ``check`` which is the same. I don't think ``fix`` will do anything since we can't commit changes to PR's comming from forks.